src/main/java/frc/robot/MoveToPoint.java

```diff
@@ -0,0 +1,94 @@
+package frc.robot;
+
+import com.ctre.phoenix6.mechanisms.swerve.SwerveRequest;
+
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.math.geometry.Pose3d;
+import edu.wpi.first.math.geometry.Rotation3d;
+import edu.wpi.first.math.geometry.Translation3d;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.Command;
+
+public class MoveToPoint extends Command {
+  /** Creates a new MoveToPoint. */
+
+  private final CommandSwerveDrivetrain driveSubsystem;
+  private final SwerveRequest.RobotCentric drive;
+  private final Timer timer = new Timer();
+
+  boolean end = false;
+  double kP = 1.3;
+  double rotationalKP = -0.05;
+  double rotationMult = 1;
+
+  int framesDropped = 0;
+
+  public MoveToPoint(CommandSwerveDrivetrain driveSubsystem, SwerveRequest.RobotCentric drive) {
+    // Use addRequirements() here to declare subsystem dependencies.
+    this.driveSubsystem = driveSubsystem;
+    this.drive = drive;
+  }
+  
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+    end = false;
+    timer.start();
+    rotationMult = 1;
+  }
+  
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {
+    double[] tagPoseRobot = LimelightHelpers.getTargetPose_RobotSpace("");
+    Pose3d pose = new Pose3d(new Translation3d(tagPoseRobot[0], tagPoseRobot[1], tagPoseRobot[2]), new Rotation3d(Math.toRadians(tagPoseRobot[3]), Math.toRadians(tagPoseRobot[4]), Math.toRadians(tagPoseRobot[5])));
+
+    if (LimelightHelpers.getTV("")) {
+      framesDropped = 0;
+    } else {
+      framesDropped++;
+      if (framesDropped > 5) {
```

Comment:
This should probably be made configurable as a parameter to the constructor. Or make the 5 a constant in the class.